client/src/partials/budget/new/new_budget.html

```diff
@@ -0,0 +1,199 @@
+                <span class="glyphicon glyphicon-plus-sign"></span> {{ "BUDGET.EDIT.CREATE" | translate }}
+              </button>
+            </h3>
+          </div>
+        </div>
+      </div>
+
+      <div class="row" ng-if="session.found">
+        <div class="panel panel-default">
+          <div class="panel-body">
+            <div class="col-xs-8">
+              <form name="budget">
+                <table class="table table-condensed form-bhima-border" summary="{{ 'EDIT.BUDGET.EDIT' | translate }}">
+                  <thead>
+                    <tr>
+                      <th colspan="5">
+                        <h4>{{ "BUDGET.EDIT.FISCAL_YEAR_LABEL" | translate }} {{ session.fiscal_year.fiscal_year_txt }}</h4>
+                      </th>
+                    </tr>
+                    <tr>
+                      <th>{{ "BUDGET.EDIT.PERIOD" | translate }}</th>
+                      <th>{{ "BUDGET.EDIT.START"  | translate }}</th>
+                      <th>{{ "BUDGET.EDIT.END"    | translate }}</th>
+                      <th>{{ "BUDGET.EDIT.BUDGET" | translate }}</th>
+                      <th>{{ "BUDGET.EDIT.FREEZE" | translate }}</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    <tr ng-repeat="budget in budgets.data | orderBy:'period_number'">
+                      <td>{{ budget.period_number }}</td>
+                      <td>{{ budget.period_start | date }}</td>
+                      <td>{{ budget.period_stop | date }}</td>
+                      <td><input type="number" 
+                           ng-model="budget.budget" 
+                           ng-change="recompute()" 
+                           ng-focus="startEditing(budget)"
+                           ng-blur="endEditing(budget)"
+                           min="0.0" 
+                           ng-disabled="session.totalBudget <= 0.0 || (session.autoAdjust && (budget.freeze && !budget.editing))"
+                           >
+                      </td>
+                      <td style="text-align: center">
+                        <input type="checkbox" 
+                         ng-model="budget.freeze">
+                      </td>
+                    </tr>
+                    <tr>
+                      <td colspan="2" style="vertical-align: middle">
+                        <b>{{ "BUDGET.EDIT.AUTO_ADJUST" | translate }}&nbsp;&nbsp;
+                        <input type="checkbox" 
+                         style="margin: 0"
+                         ng-disabled="session.totalBudget <= 0.0"
+                         ng-change="recompute()"
+                         ng-model="session.autoAdjust"></b>
+                      </td>
+                      <td style="vertical-align: middle">
+                        <span class="pull-right">{{ "BUDGET.EDIT.TOTAL" | translate }}</span>
+                      </td>
+                      <td style="vertical-align: middle">
```

Comment:
You have used `style="vertical-align:middle"` three times in this table on `<td>`s.  Could this be applied to the `<tr>` or make into a class that does this to avoid repetition?